### ISSUE TITLE
`<x:fill />` should have a `x:patternFill` or `x:gradientFill` child

### ISSUE THREAD

**Issue by muelli** [ONLYOFFICE/DocumentServer#1688]
**Do you want to request a *feature* or report a *bug*?**

I come from https://foss.heptapod.net/openpyxl/openpyxl/-/issues/1805 which investigated that
the specification is clear.
```
<xsd:complexType name="CT_Fill">
  <xsd:choice minOccurs="1" maxOccurs="1">
    <xsd:element name="patternFill" type="CT_PatternFill" minOccurs="0" maxOccurs="1"/>  
    <xsd:element name="gradientFill" type="CT_GradientFill" minOccurs="0" maxOccurs="1"/>
  </xsd:choice>
</xsd:complexType>
```

But the fills section of the stylesheet is
```
<x:fills count="7" xmlns:x="http://schemas.openxmlformats.org/spreadsheetml/2006/main">
  <x:fill>
    <x:patternFill patternType="none" />
  </x:fill>
  <x:fill>
    <x:patternFill patternType="gray125" />
  </x:fill>
  <x:fill>
    <x:patternFill patternType="solid">
      <x:fgColor rgb="FFFFC7CE" />
      <x:bgColor indexed="31" />
    </x:patternFill>
  </x:fill>
  <x:fill>
    <x:patternFill patternType="solid">
      <x:fgColor rgb="FFFFC7CE" />
      <x:bgColor rgb="FFFFC7CE" />
    </x:patternFill>
  </x:fill>
  <x:fill>
    <x:patternFill patternType="solid">
      <x:fgColor rgb="FFF2F2F2" />
      <x:bgColor rgb="FFF2F2F2" />
    </x:patternFill>
  </x:fill>
  <x:fill>
    <x:patternFill patternType="solid">
      <x:fgColor theme="0" tint="0" />
      <x:bgColor theme="0" tint="0" />
    </x:patternFill>
  </x:fill>
  <x:fill />
</x:fills>
```
So, the final empty Fill violates the specification.

**What is the current behavior?**

Empty fills are created

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

there's a sample document in the other bug report

**What is the expected behavior?**

documents in accordance with the spec

**Did this work in previous versions of DocumentServer?**

I don't know, but there is https://github.com/ONLYOFFICE/DocumentServer/issues/682 which claims that this has been fixed already.

**DocumentServer version:**

7.0.1

**Comment by ShockwaveNN** [comment]
Could you describe a step-by-step scenario of how to create file file with <x:fill /> from empty file?

Are you created in from new empty file, converted some file in other format or for example using DocumentBuilder?

**Comment by ShockwaveNN** [comment]
Ok, in that case, I'm not sure that our developers will be able to find a source of the problem since maybe this file was created in some older version of DocumentServer, and even if you open it and save it in a newer version - it carefully 'do not touch' anything that wasn't changed in the file in editor.

Our developers will look at this file as soon as they got some free time, but not sure if file like those can be created in current stable version

**Comment by ShockwaveNN** [comment]
The problem is fixed via https://github.com/ONLYOFFICE/sdkjs/commit/46ee8c587b212898f2069ea15f74117f5fab1c7c and will be available as DocumentServer v7.1, but currently, we have no ETA date

**Comment by ShockwaveNN** [comment]
DocumentServer v7.1.0 is released. This issue should be fixed

I'll close this issue, but feel free to comment or open new issues if there are some troubles
